packages/connect/src/device/Device.ts

```diff
@@ -116,7 +116,11 @@ type DeviceParams = {
 export class Device extends TypedEmitter<DeviceEvents> {
     public readonly transport: Transport;
     public readonly transportPath;
-    public readonly bluetoothProps;
+    public readonly bluetoothProps: {
+        // TODO: type it properly.
+        channels: any;
```

Comment:
I think it can stay part of bluetoothProps for now, but lets keep in mind that in the future we will want to open up this feature for usb too. So maybe if we come up with a better place..